cnv/cnv_users_guide/cnv-fencing-unhealthy-nodes.adoc

```diff
@@ -0,0 +1,27 @@
+[id="cnv-fencing-unhealthy-nodes"]
+= Fencing unhealthy nodes
+include::modules/cnv-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: cnv-fencing-unhealthy-nodes
+toc::[]
+
+Fencing is the process of isolating a failed node to protect a cluster and its resources.
+A node is considered 'fenced' when it can no longer access shared resources in order
+for those resources to be used elsewhere. Without fencing, a failed node can result in data corruption.
+
+In the case of an unhealthy node, Pods and virtual machines that depend on
+persistent storage cannot be replaced on another node until it can be confirmed
+that the original Pod or virtual machine is no longer accessing the storage.
```

Comment:
I think this sentence should be broken up into shorter pieces and rewritten so that it uses the active voice.

Reply:
I've had a good hard crack at it but it is a complex concept and I haven't been able to change it to make it simpler or more active (it's a passive process from the user POV and in a somewhat hypothetical context). 
For sanity I workshopped it with some other writers (one of whom worked on fencing for OSP) and that was also the consensus.